docs/governance/politeia/example-proposals.md

```diff
@@ -0,0 +1,156 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Politeia.svg" /> Example Politeia Proposals
```

Comment:
I don't think we should be putting the proposal example text in codeblocks on this page. They aren't ideal for prose as they use the monospace font and they don't format nicely on small screens/windows. 

That being said, we don't have many alternatives. The "correct" approach is probably to use blockquotes, but this would make the text green as a result of #566. I just tried this out and it doesn't look right.

Lets keep the codeblocks for now and I will open a new issue for this: #759

Reply:
Yeah, just copied that mindlessly from the markdown embedded in the onboarding.js file. 

I will say that, while the light green isn't great, I do like the spash of color. It makes the docs look more professional and modern. Perhaps we can find better ways to enhance design elements.